velox/docs/functions/iceberg/bucket.rst

```diff
@@ -0,0 +1,8 @@
+=============================
+Bucket Function
```

Comment:
Since there are only 6 functions in total, let's put them all together. Let's not create a separate page for each function.